ONSdigital/rm-tools
# Changed go_live date to the future

# Motivation and Context
We found an acceptance test was failing, when it was looking navigating to a collection exercise, which should be set to `ready_for_love`, unfortunately the date originally set was today, so the status was set to `live`. Hence failing tests

# What has changed
We have set the `go_live` date in the future for now, with a ticket being added in the backlog to improve the test to be smarter.

# How to test?
Pull this branch then in the Acceptance tests `setup_data.sh` add this code after the `else` on line 9:
```
    git clone ${rm_tools_repo_url} tmp_rm_tools;
    cd tmp_rm_tools;
    git checkout fix-breaking-at-with-ready-for-live-data;
    cd ..;
```
Then run AT's, should all pass.
